qemu/tests/blockdev_commit_with_ioerror.py

```diff
@@ -0,0 +1,94 @@
+from aexpect import ShellTimeoutError
+
+from virttest import utils_numeric
+from virttest import utils_disk
+
+from provider import backup_utils
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitWithIoerror(BlockDevCommitTest):
+
+    def generate_tempfile(self, root_dir, filename, size="1000M", timeout=20):
+        """Generate temp data file in guest"""
+        self.session = self.main_vm.wait_for_login()
+        file_path = "%s/%s" % (root_dir, filename)
+        count = int(
+            utils_numeric.normalize_data_size(
+                size,
+                order_magnitude="M",
+                factor=1024))
+        dd_cmd = self.main_vm.params.get(
+            "dd_cmd", "dd if=/dev/urandom of=%s bs=1M count=%s oflag=direct")
```

Comment:
I think we can use count=1000 directly here, for we are using 1000 in the dd_done msg.